### DISCUSSION TITLE
Did the Cosmo html theme change colors?

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#5751] [Q&A] [answered]
### Description

Did the Cosmo theme colors change?

I have this theme in my blog and the navigation bar was a kind of dodgerblue color and now is a different color (much ligther tone).

**Comment by mcanouil** [comment, marked as answer]
Yes, see #5521 for how to "revert".

That change was made in [v1.3.266](https://github.com/quarto-dev/quarto-cli/releases/tag/v1.3.266)

**Comment by ManuelSpinola** [comment]
Thank you very much.
